src/Extension/FluentExtension.php

```diff
@@ -845,6 +869,30 @@ public function LocaleInformation($locale = null)
         return RecordLocale::create($this->owner, $localeObj);
     }
 
+    /**
+     * Get list of locales where record is localised in
+     *
+     * @return array
+     */
+    public function LocaleInstances()
```

Comment:
The master branch of fluent requires PHP 7.1 already, would you like to add some return types?